R/getEnvironmentDS.R

```diff
@@ -0,0 +1,39 @@
+#'@name getEnvironmentDS
+#'@title retrieves an environment from its name.
+#'@description retrieves an environment from its name. It verifies the environment exists within the specified parent environment, before returning the environment itself.
+#'@details The helper function verifies a environment exists on the search path.By default, the parent environment used is the ".GlobalEnv".
+#'@param environment.name the name of an environment available on a DataSHIELD server.
+#'@param package.name the name of the package as appearing in \code{\link{search}}
+#'@return
+#'\itemize{
+#' \item  an empty enviroment if no environment is found.
+#' \item  the reference to the environment itself.
+#'}
+#'@details The helper function verifies a environment exists, within another environment  on a dataSHIELD server. By defautl the parent environment used is the
+#'\code{globalenv()}. For testing and other purposes, other parent environment can be specify
+#'@keywords
+#'@author P.Ryser-Welch, Newcastle University, DataSHIELD team
+#
+
+getEnvironmentDS <- function(environment.name = NULL, package.name = ".GlobalEnv")
```

Comment:
Just a quick note (not sure it is relevant): this reference to .GlobalEnv would not work with DSLite